src/test/java/org/traccar/geocoder/GeocoderTest.java

```diff
@@ -41,7 +41,7 @@ public void testGisgraphy() {
     @Test
     public void testOpenCage() {
         Geocoder geocoder = new OpenCageGeocoder(
-                "http://api.opencagedata.com/geocode/v1", "SECRET", 0, new AddressFormat());
+                "http://api.opencagedata.com/geocode/v1", "SECRET", "en", 0, new AddressFormat());
```

Comment:
I think we should test with `null`, like other geocoders.

Reply:
well ideally you would have many tests with all possible values, but sure, whatever you want